src/datachain/data_storage/metastore.py

```diff
@@ -1593,6 +1599,18 @@ def list_jobs_by_ids(self, ids: list[str], conn=None) -> Iterator["Job"]:
         query = self._jobs_query().where(self._jobs.c.id.in_(ids))
         yield from self._parse_jobs(self.db.execute(query, conn=conn))
 
+    def get_last_job_by_name(self, name: str, conn=None) -> Optional["Job"]:
```

Comment:
Let's please move this implementation into `SQLiteMetastore` class?
Just to be on a safe side and be sure this logic will not be run in SaaS?